Sources/Client.swift

```diff
@@ -234,58 +311,58 @@ extension Client {
     /**
      Fetch a single Asset from Contentful
 
-     - parameter id: The identifier of the Asset to be fetched
-     - parameter completion: A handler being called on completion of the request
+     - Parameter id: The identifier of the Asset to be fetched
```

Comment:
From here on you seem to not use a full stop at the end of the parameter line anymore. Which one would you like to continue using? (with or without `.` at the end?)